### ISSUE TITLE
about line break rule

### ISSUE THREAD

**Issue by gzliudan** [prettier-solidity/prettier-plugin-solidity#630]
If a function have more than 2 parameters or more than 2 return values, then it will be breaked into serveral lines. Is there a new rule which will not break line, or only break line according to line length(printWidth)? 

```solidity
function getName(uint256 a, uint256 b, uint256 c) external view returns (uint256 d, uint256 e, uint256 f);
```

becomes:

```solidity
function getName(
        uint256 a,
        uint256 b,
        uint256 c
    )
        external
        view
        returns (
            uint256 d,
            uint256 e,
            uint256 f
        );
```

**Comment by mattiaerre** [comment]
I think this is by design, you may want to try to skip specific lines maybe? cc @fvictorio

**Comment by pofulu** [comment]
Really need a rule to control the parameter count. Have any ideas?

**Comment by Janther** [comment]
This has been addressed as we moved into v1.0.0
